### PR TITLE
Fancy README table; fix rubinius build

### PR DESCRIPTION
**Pull request by apurvis**
@kamui fancy fancy

### CODE REVIEW

**Comment by apurvis** [comment]
i ordered these according to my judgement of relevance to people reading the docs...  maybe they should just be alphabetical?

File: spec/exponential_backoff_spec.rb [hunk 1]
```diff
@@ -26,53 +26,29 @@
...
-      1.2406087918999114,
-      1.7632403621664823,
-      2.338001204738311,
-      4.350816718580626,
-      5.339852157217869,
-      11.889873261212443,
-      18.756037881636484,
-    ])
+    expect(subject.new(tries: 9).intervals.map { |x| x.round(3) }).must_equal(
+      [0.524, 0.911, 1.241, 1.763, 2.338, 4.351, 5.34, 11.89, 18.756]
```
Comment (apurvis) [inline spec/exponential_backoff_spec.rb]: Apparently rubinius has totally different floating point noise... so the build fails on the way out digits

**Comment by apurvis** [comment]
@kamui check out the rubinius section of the travis build.  not only does it have wildly different ideas about floating point noise (corrected in this PR) it seems it is flat out refusing to build the kernel extensions.

up to you whether it's worth addressing these issues or just dropping rubinius support.

**Comment by kamui** [comment]
rbx was always in the `allowed_failures` travis section because I considered it best effort. I'll keep tracking newer rbx versions and maybe one day the specs will pass again.

So, you can drop the rbx support changes in this PR and just leave in the README changes. It looks good to me.

**Comment by apurvis** [comment]
pretty sure there's 0 chance the specs will ever pass without these changed given the floating point noise issue, but ok.  at least now it actually runs the specs, whereas before they didn't even run.

**Pull request closed without merging**